website/docs/api/docusaurus.config.js.mdx

```diff
@@ -767,9 +767,9 @@ export default {
 
 ### `headTags` {#headTags}
 
-An array of tags that will be inserted in the HTML `<head>`. The values must be objects that contain two properties; `tagName` and `attributes`. `tagName` must be a string that determines the tag being created; eg `"link"`. `attributes` must be an attribute-value map.
+An array of tags that will be inserted in the HTML `<head>`. The values must be objects that contain two properties; `tagName` and `attributes`. `tagName` must be a string that determines the tag being created; eg `"link"`. `attributes` must be an attribute-value map. When custom html elements are needed, set `customElement: true`.
 
-- Type: `{ tagName: string; attributes: Object; }[]`
+- Type: `({ tagName: string; attributes: Object; } | { tagName: string; attributes?: Object; customElement: true; })[]`
```

Comment:
I'm going to avoid the union type in docs. Although more accurate, it could make the docs harder to read for the usual case where devs are not using custom elements

Although we can be flexible, I'm not sure what could be the value of adding a custom element in head if it doesn't have any attribute. It would only be a "marker" somehow, but it's not really your use-case. I think it's fine if we keep documenting it as if the attributes were required (the object can be empty anyway)